features.md

```diff
@@ -0,0 +1,16 @@
+#### Features:
+
+  1. Search and Find Events
+    * Displays events based on GPS location and interests
+    * Gives the user the possibility to apply filters (as event topic, type, location, time, organiser etc)
+    * Notiication Alerts when events match your search criteria
```

Comment:
@jsha2007 - Can you edit this line to correct the spelling of "Notification" alerts